app/components/appMainModule/committeePageModule/committeePageRoutes.js

```diff
@@ -44,7 +44,7 @@ function committeePageRoutes($stateProvider) {
     template: require('./committeePage.html'),
     ncyBreadcrumb: {
       label: '{{ committee.name }}',
-      parent: 'appMain.locality'
+      parent: 'appMain'
```

Comment:
Yeah, I don't even think `appMain.locality` is a real state anymore (we're using `appMain.localePage`). You're right, I'm stumped on why the breadcrumbs break now.